src/renderer/components/dialogs/TemplateProjectDialog.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+  <div v-show="visible === true" class="dialog">
+    <div class="content">
+      <p class="heading">Create New SpringRoll Project</p>
+
+      <div class="options">
+        <div class="projectLocation">
+          <p class="projectLocation-heading">New Project Location</p>
+          <div class="projectLocation-content">
+            <input
+              class="urlInput"
+              :value="projectLocation"
+            />
+            <button
+              class="dirBrowseBtn"
+              @click="onProjectLocationBtnClick()"
+            >
+              Browse
+            </button>
+          </div>
+        </div>
+
+        <div class="templateType">
+          <p class="templateType-heading">Template Type</p>
+          <div class="templateType-content">
+            <div class="templateOption">
+              <input
+                id="pixiOption"
+                type="radio"
+                name="templateType"
+                checked
+                @change="setTemplateType('pixi')"
+              />
+              <label for="pixiOption">PIXI</label>
+            </div>
+
+            <div class="templateOption">
+              <input
+                id="phaserOption"
+                type="radio"
+                name="templateType"
+                @change="setTemplateType('phaser')"
+              />
+              <label for="phaserOption">Phaser 3</label>
+            </div>
+
+            <div class="templateOption">
+              <input
+                id="createjsOption"
+                type="radio"
+                name="templateType"
+                @change="setTemplateType('createjs')"
+              />
+              <label for="createjsOption">CreateJS</label>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="actions">
+        <button
+          id="cancelBtn"
+          @click="onBtnCancelClick()"
+        >
+          Cancel
+        </button>
+        <button
+          id="confirmBtn"
+          @click="onBtnConfirmClick()"
+        >
+          Confirm
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import { remote } from 'electron';
+
+export default {
+  props: {
+    // Variables
+    visible: {
+      type:Boolean,
+      default: false
+    },
+
+    // Callbacks
+    onCancel: {
+      type: Function,
+      default: () => {}
+    },
+    onConfirm: {
+      type: Function,
+      default: () => {}
+    }
+  },
+
+  /**
+   * Data object
+   */
+  data: function() {
+    return {
+      templateType: 'pixi'
```

Comment:
Surprised you chose pixi as the default 🙃

Reply:
:D Pixi seems the more popular option from what I've seen